### ISSUE TITLE
the logic of the code

### ISSUE THREAD

**Issue by stayliv3** [everyonesdesign/OpenSearchInNewTab#1]
everyonesdesign:

hello,i want to use the function in my own plugin, i have the same need for open a new tab for search result .I can not understand your code though it is very simple .can u explain something for me .

thanks.

**Comment by everyonesdesign** [comment]
@stayliv3 
OK, no problem, I'll try to explain it.

Firstly, how sublime search tab opening works (I found out it experimentally):
- the editor looks for a tab with name 'Find Results'
- if it's found then use it to append new search results
- if there's no such a tab then create it

So I decided to rename the tab to trick the Sublime.
I change tab's name to 'Find Results ' (with space as it reads the same by human but it's not the same thing for the editor) on `on_activated` hook:

``` python
def on_activated(self, view):
        if view.name() == DEFAULT_NAME: # DEFAULT_NAME == 'Find Results'
            view.set_name(ALT_NAME) # ALT_NAME == 'Find Results '
```

That's how it's basically work.

Then I found out that other plugins (in my case it was BetterFindBuffer) also detected search tab by its name so I decided to set back name 'Find Results' on `on_text_command` hook so the plugins can execute their text command on the correctly named tab; After that execution `post_text_command` again sets it to name with a space.

I see that it's not 100% reliable but that's the best way I've figured out to do that.
